cxx/test_schema/with_noisy.schema

```diff
@@ -0,0 +1,6 @@
+clean bernoulli R1 D1
+noisy gaussian R2 R6 D1 D2
```

Comment:
This is making me realize that we may want to rename gaussian or normal? It's confusing to have synonyms for the same distribution mean the noisified and clean version.